qa/rpc-tests/test_framework/util.py

```diff
@@ -603,6 +604,10 @@ def split_transaction(node, prevouts, toAddrs, txfeePer=DEFAULT_TX_FEE_PER_BYTE,
     finally:
       decimal.getcontext().prec = decContext
 
+def assert_not_equal(thing1, thing2):
```

Comment:
I need this one for the PV regression test to work...it's not used by any other script.